### ISSUE TITLE
GET contacts: Cannot read properties of undefined (reading 'map')

### ISSUE THREAD

**Issue by glbatistabh** [pedroslopez/whatsapp-web.js#1507]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

```
--GET contacts ERRO Error: Evaluation failed: TypeError: Cannot read properties of undefined (reading 'map')
    at Object.window.WWebJS.getContacts (__puppeteer_evaluation_script__:381:25)
    at __puppeteer_evaluation_script__:2:34
    at ExecutionContext._evaluateInternal (/root/whats-api-test/node_modules/puppeteer/lib/cjs/puppeteer/common/ExecutionContext.js:221:19)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async ExecutionContext.evaluate (/root/whats-api-test/node_modules/puppeteer/lib/cjs/puppeteer/common/ExecutionContext.js:110:16)
    at async Client.getContacts (/root/whats-api-test/node_modules/whatsapp-web.js/src/Client.js:658:24)
```

### Expected behavior

return a contact list

### Steps to Reproduce the Bug or Issue

```
client.getContacts().then((contacts) => {
  res.status(200).json({contacts: _.sortBy(contacts, 'name')});
}).catch((error) => {
  console.log("--GET contacts ERRO", error);
  res.status(402).json({contacts: null, status: 'error', message: error});
});
```

### Browser Type

Chromium

### WhatsApp Account Type

Standard

### Does your WhatsApp account have multidevice enabled?

Yes, I am using Multi Device

### Environment

debian 10.12
node v16.15.0
Phone Android
whatsapp-web.js  1.16.6
WATSAPP version: v2.2220.8

**Comment by Yuri-Lima** [comment]
Go check #1503 that get it fixed. Hope to be merged soon.

**Comment by pedroslopez** [comment]
Fixed in #1503. Thanks for reporting, please update to v1.16.7!
